selfdrive/car/tests/routes.py

```diff
@@ -17,6 +17,7 @@
 # TODO: add routes for these cars
 non_tested_cars = [
   FORD.F_150_MK14,
+  FORD.MUSTANG_MACH_E_MK1,
```

Comment:
Is there any route for this car?

Reply:
I'll get routes added for the Mach E and the F150 in the next few hours.

Reply:
Added both F150 ICE and Mach E routes